pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/InTransformFunction.java

```diff
@@ -365,11 +380,56 @@ public int[] transformToIntValuesSV(ValueBlock valueBlock) {
             }
           }
           break;
+        case UNKNOWN:
+          break;
         default:
           throw new IllegalStateException();
       }
     }
 
     return _intValuesSV;
   }
+
+  @Nullable
+  @Override
+  public RoaringBitmap getNullBitmap(ValueBlock valueBlock) {
+    int length = valueBlock.getNumDocs();
+    RoaringBitmap result = new RoaringBitmap();
+    RoaringBitmap mainFunctionNullBitmap = _mainFunction.getNullBitmap(valueBlock);
+    if (mainFunctionNullBitmap != null) {
+      result.or(mainFunctionNullBitmap);
+    }
```

Comment:
Return early if nullbitmap cardinality is equal to num of docs?

Reply:
Done.